### ISSUE TITLE
Creating an open access archive for research on men and masculinities.

### ISSUE THREAD

**Issue by DrPeterFerry** [sparcopen/doathon#29]
[//]: # "=======================
Even if you know Github well, we suggest you read this. Anything between these lines you can leave or delete, as they won't display anyway when you post (you can check this via Preview changes). They're here to help you complete issues quickly and in a way that will help other participants. If you're posting a new project, or challenge. We suggest you fill out the Google Forms first.
============================"

## At a glance

[//]: # "=======================
Creating an open access archive for research on men and masculinities, with particular focus on educating men and boys on issues of gender equality
============================"

* **Submission Name:** Open Access Archive for research on men and masculinities, with particular focus on educating men and boys on issues of gender equality
* **Contact Lead:** @DrPeterFerry or peter.ferry@uis.no
* **Region**: Worldwide aim but based in Norway
* **Issue Area:** #OpenAccess, #OpenData, #OpenEducation, #OpenResearch
* **Issue Type:** #Project
* **Types of Support Needed:**  #Advocacy_and_Policy, #Coding, #Communications, #EventPlanning, #Research

## Description
[//]: # "=======================

The main aim of my time at the OpenCon 2017 conference is to kickstart my Open Access project to create an Open Access Archive for research on men and masculinities. The primary drive behind this project is to offer an free open space for men and boys to have access to cutting edge research (primarily from academia) that deals with a range of issues linked to men and masculinities and gender equality.
============================"

##  How can others contribute?
[//]: # "======================= 
What I want from my time at OpenCon is:
 - advice and guidance on how to set up on online archive
- strategies for contacting researchers to suggest their involvement
- warnings of the typical obstacles that I will need to overcome
- help on how to promote the archive, both to researchers as well as other institutions
- ideas about funding
- tips on how to maximise the reach of the archive

Plus any other ideas on what I should be considering when setting up such an archive.

Please feel free to contact me on Twitter @DrPeterFerry or my email peter.ferry@uis.no

Looking forward to working with you!

 ============================"

[//]: # "=======================
You're ready to post!!! After posting your issue, the real work begins. Next, you might want to:
   Tweet a link to this issue with #opencon so others can join in
   Make another issue to involve people in your work - remember to use your metadata
   Come back from time to time and update the community on your project.
You'll get an email update whenever someone interacts with your issue.
============================"

_This post is part of the OpenCon 2017 Do-A-Thon. Not sure what's going on? [Head here](http://doathon.opencon2017.org/)._

**Comment by DrPeterFerry** [comment]
@lorrainechu3n 
Hi Lorraine, I am planning to join Zoe and the Humanities guys in the do-a-thon: "Open Humanities & Social Sciences Do-a-thon Session: Out of Sunday's unconference came several ideas for action - so let's make them happen!". Shall I just leave this open here?

**Comment by lorrainechu3n** [comment]
Yes, of course, no problem @DrPeterFerry — just be clear on how folks can contribute and I would suggest watching the Github issue for any questions or comments :)
